cmd/agent/app/reporter/grpc/reporter.go

```diff
@@ -71,7 +73,12 @@ func (r *Reporter) send(ctx context.Context, spans []*model.Span, process *model
 	req := &api_v2.PostSpansRequest{Batch: batch}
 	_, err := r.collector.PostSpans(ctx, req)
 	if err != nil {
-		r.logger.Error("Could not send spans over gRPC", zap.Error(err))
+		stat, ok := status.FromError(err)
+		if ok && stat.Code() == codes.PermissionDenied && stat.Message() == "missing tenant header" {
+			r.logger.Debug("Could not send spans over gRPC", zap.Error(err))
```

Comment:
It's worth distinguishing the error messages a bit more, not just by the error level.